src/reducers/businessCaseReducer.ts

```diff
@@ -37,6 +41,25 @@ function businessCaseReducer(
         ...state,
         isLoading: false
       };
+    case postBusinessCase.SUCCESS:
+      return {
+        ...state,
+        form: prepareBusinessCaseForApp(action.payload)
+      };
+    case storeBusinessCase.TRIGGER:
+      return {
+        ...state,
+        form: {
+          ...state.form,
+          ...action.payload
```

Comment:
Is there concern here if the user keeps editing out the form while it's being saved? When that save payload is returned, we need the user's changes to take precedence. Will they?

Reply:
Good point. I don't think this should be an issue because form state is persisted with Formik and not Redux. It's a bit duplicative, but I'm not actually sure this will overwrite as the user is filling out the form.

It's mainly used to store the business case ID after the create, but we can do it a little differently.